### DISCUSSION TITLE
잠시후 다시시도하세요 1시간뒤 해제된듯하나

### DISCUSSION THREAD

**Discussion by koouoo722** [changdoc/naver-vaccine-macro#59] [Q&A]
다시 자동예약을 진행이 가능하여 진행했더니
재차단이 걸려서 다시 1시간 차단이 되었네요ㅠㅠ

공지를 보고 병원2개를 오전 1개 종료후 오후 1개 이렇게 진행했는데
그래서 문제가 된건지 다시 차단이 되서 하하하..

이건 네이버측이 진행한거라 방법이 없겠죠...?

**Comment by OneTop4458** [comment]
https://github.com/changdoc/naver-vaccine-macro/discussions/51

참고 바라며 지속 발생 시 인터벌은 2500ms로 조절 후 시도 바랍니다.
